R/recall_count.R

```diff
@@ -1,12 +1,14 @@
 #' count_recall_by_make
 #'
-#' @param make list string
-#' @param manufacturer A bool, with this flag we are asking for the manufacturer not the make
-#' @param start_year An integer
-#' @param end_year An integer
+#' Queries the website with a url that it generates from it's parameters.
+#'
+#' @param make List string, name(s) of the vehicle(s).
+#' @param manufacturer A boolean, with this flag we are asking for the manufacturer not the make.
+#' @param start_year An integer, start of year range, defaults to 1900.
+#' @param end_year An integer, end of year range, defaults to 2100.
 #' @param api_key A string, optional
 #'
-#' @return integer
+#' @return A integer, the number of recalls for input make in given year range.
 #' @export
 #'
 #' @examples
```

Comment:
should have a couple examples per function, all wrapped in `\dontrun{}` - there's a discrepancy in https://github.com/ThinkR-open/prepare-for-cran which says to use `\donttest{}` vs R Packages saying to use `\dontrun{} and not `\donttest{}`. I suggest we go with `\dontrun{}` as i've seen more examples of packages that use this